### DISCUSSION TITLE
Freezing screen

### DISCUSSION THREAD

**Discussion by IvoBastos** [IvoBastos/PSR-Augmented-Reality-Paint#6] [Q&A]
Aparentemente, ao correr o programa, assim que pressiono a tecla 's' na tentativa de desenhar, o meu ecrã congela e a única solução é dar quit ao programa. Mais alguém tem este problema? Quando reunimos pareceu estar tudo a funcionar conforme suposto no pc do Jorge.

**Comment by JorgeFernandes-Git** [comment]
Sim testem pff, pode ter ficado alguma coisa mal do commit de hoje
